### PR TITLE
Cache minified content for performance

### PR DESCRIPTION
**Pull request by artursvonda**
Improves performance significantly.

### CODE REVIEW

**Comment by robertfausk** [comment]
+1

File: Twig/Extension/JshrinkExtension.php [hunk 1]
```diff
@@ -25,17 +26,33 @@ class JshrinkExtension extends Twig_Extension
...
      * Twig extension based on bundle configuration.
      *
-     * @param array $config
-     * @param bool  $enabled
+     * @param CacheInterface $cache
+     * @param array          $config
+     * @param bool           $enabled
      */
-    public function __construct(array $config = array(), $enabled = true)
+    public function __construct(CacheInterface $cache, array $config = [], $enabled = true)
```
Comment (robertfausk) [inline Twig/Extension/JshrinkExtension.php]: ```array $config = []``` should be ```array $config = array()``` to be compatible with supported php version >= 5.3.2

**Comment by artursvonda** [comment]
Fixed the array shorthand. Thanks @robertfausk for PR, I just made the changes in original commit to keep the history cleaner.

**Comment by EmanueleMinotto** [comment]
Hi @artursvonda, sorry for the long delay but I missed the notification on 9 Mar ^^'

A test fails (launched with `./vendor/bin/phpunit`):
```
1) Salva\JshrinkBundle\Tests\Twig\Extension\JshrinkFilterTest::testProdScripts
Twig_Error_Runtime: The "native_profiler" extension is not enabled in "@SalvaJshrinkBundle/Tests/Resources/views/assetic.html.twig".

SalvaJshrinkBundle/vendor/twig/twig/lib/Twig/Environment.php:682
SalvaJshrinkBundle/vendor/twig/twig/lib/Twig/Template.php:313
SalvaJshrinkBundle/vendor/twig/twig/lib/Twig/Template.php:287
SalvaJshrinkBundle/vendor/twig/twig/lib/Twig/Template.php:298
SalvaJshrinkBundle/vendor/twig/twig/lib/Twig/Environment.php:293
SalvaJshrinkBundle/Tests/Assetic/Filter/JshrinkFilterTest.php:49

2) Salva\JshrinkBundle\Tests\Twig\Extension\JshrinkExtensionTest::testIntegration with data set #0 ('tags/jshrink.test', '"jshrink" tag', '', array('\n<script>{% jshrink %}\n    /*...cript>'), false, array(array('--DATA--\nreturn array()\n--EXP...cript>', '\nreturn array()\n', '', '\n<script>/*! copyright */\n$(d...cript>')))
Argument 1 passed to Salva\JshrinkBundle\Twig\Extension\JshrinkExtension::__construct() must implement interface Salva\JshrinkBundle\Cache\CacheInterface, none given, called in SalvaJshrinkBundle/Tests/Twig/Extension/JshrinkExtensionTest.php on line 19 and defined

SalvaJshrinkBundle/Twig/Extension/JshrinkExtension.php:42
SalvaJshrinkBundle/Tests/Twig/Extension/JshrinkExtensionTest.php:19
SalvaJshrinkBundle/vendor/twig/twig/lib/Twig/Test/IntegrationTestCase.php:84
SalvaJshrinkBundle/vendor/twig/twig/lib/Twig/Test/IntegrationTestCase.php:28
```

**Comment by artursvonda** [comment]
@EmanueleMinotto, no worries. Pushed a fix for the second test. Didn't get to the first test. Don't think it's related to my changes. It something that came up only when I upgraded to latest version of dependencies.

**Comment by EmanueleMinotto** [comment]
Great, PR merged.
Thank you @artursvonda for your work. :)

**Pull request merged**